### PR TITLE
Tests for fulltext queries

### PR DESCRIPTION
**Pull request by mgencur**
The TestFullTextTermRightOperandAnalyzed test is failing for me and I have no idea why:-/ The rest is passing.

### CODE REVIEW

**Comment by mgencur** [comment]
Alright, so the failing test must be caused by this issue: https://issues.jboss.org/browse/ISPN-7301 
My tests were using ISPN Beta1 and this is fixed in Beta2 so the test should start passing with Beta2

File: src/test/cs/Infinispan/HotRod/RemoteFulltextQueryTest.cs [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+            Assert.AreEqual(1, transactions.Count);
+        }
+
+        [Test]
+        public void TestFullTextTermRightOperandAnalyzed()
+        {
+            IRemoteCache<String, Transaction> transactionCache = remoteManager.GetCache<String, Transaction>(NAMED_CACHE);
+
+            QueryRequest qr = new QueryRequest();
+            qr.QueryString = "from sample_bank_account.Transaction t where longDescription:'RENT'";
```
Comment (anistor) [inline src/test/cs/Infinispan/HotRod/RemoteFulltextQueryTest.cs]: Keep it upper case to prove it works regardless of the case after fixing ISPN-7301. Lower case was already working before that.

Comment (rigazilla) [inline src/test/cs/Infinispan/HotRod/RemoteFulltextQueryTest.cs]: We need an [Ignore] attribute in the meanwhile

**Comment by mgencur** [comment]
Added Ignore attribute.

**Comment by rigazilla** [comment]
merged, thanks!

**Pull request merged**